docs/current_docs/manuals/developer/services.mdx

```diff
@@ -192,9 +288,27 @@ If you're not interested in what's happening in the background, you can skip thi
 
 Consider this example:
 
-```typescript file=./snippets/services/service-lifecycle-1/index.ts
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./snippets/services/service-lifecycle-1/go/main.go
+```
+
+</TabItem>
+<TabItem value="Python">
+
+```python file=./snippets/services/service-lifecycle-1/python/main.py
+```
+
+</TabItem>
+<TabItem value="TypeScript">
+
+```typescript file=./snippets/services/service-lifecycle-1/typescript/index.ts
 ```
 
+</TabItem>
+</Tabs>
+
 Here's what happens on the last line:
 
 1. The client requests the `ping` container's stdout, which requires the container to run.
```

Comment:
I believe this block of text should be within the tabs, as it differs based on the code. See for example this screenshot of the Python version from the live docs.

![image](https://github.com/dagger/dagger/assets/112123850/5ebdebce-f752-45f7-aa13-5e44a5ca4fe6)

Reply:
Good catch, thanks

Reply:
@vikram-dagger fixed!